### PR TITLE
[READY] Digitigrade shoes

### LINKED ISSUE #11: Digitigrade support I guess
Digitigrade shoes/suits. support the lizards properly. Reference old code to figure it out if you need help?

### PR DESCRIPTION
**Pull request by CyoTheProtogen**
shoesies
(Same picture used in the old PR, but literally nothing changed sprite wise.)
## About The Pull Request
Ports over digitigrade shoes from the old repo, while also making it a bit more modular.

## Why It's Good For The Game
Special lizards finally get their special shoes!

Resolves #11

## Changelog
:cl:
add: Ports digitigrade shoes from the old repo!
add: Digitigrade magboots that are available in the "Advanced Engineering" tech node
add: Digitigrade jumpboots in the mining point vendor for 2500 points (Same as regular jumpboots)
tweak: digi shoes are now available in slightly more vendors and closets, just a little QoL.
balance: captain now starts with regular brown digi shoes instead of the digi combat boots since its free armour.
balance: digi combat boot price in the mining point vendor was increased to 450 for the reasons stated above.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1023,9 +1023,11 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		if(ITEM_SLOT_FEET)
 			if(H.num_legs < 2)
 				return FALSE
-			if(DIGITIGRADE in species_traits)
+//			if(DIGITIGRADE in species_traits)
+			if((DIGITIGRADE in species_traits) == !(I.flags_inv & FULL_DIGITIGRADE)) //fulp edit oh god oh fuck
 				if(!disable_warning)
-					to_chat(H, "<span class='warning'>The footwear around here isn't compatible with your feet!</span>")
+//					to_chat(H, "<span class='warning'>The footwear around here isn't compatible with your feet!</span>")
+					to_chat(H, "<span class='warning'>These shoes aren't compatible with your feet!</span>") //fulp edit 2, electric boogaloo
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/carbon/human/species.dm:1030]: Was this edit necessary?

File: fulp_modules/fulp_clothing/outfits/digi_lizard.dm [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		if("Psychologist")
+			newShoes = new /obj/item/clothing/shoes/laceup/digitigrade
+
+		// Civilian
+		else
+			newShoes = new /obj/item/clothing/shoes/sneakers/brown/digitigrade
+
+	// Deletes the stinky non-digitigrade shoes (if for some reason they spawn with them)
+
+    if (H.shoes)
```
Comment (JohnFulpWillard) [inline fulp_modules/fulp_clothing/outfits/digi_lizard.dm:99]: This is using spaces instead of tabs

File: code/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (CyoTheProtogen) [reply code/modules/mob/living/carbon/human/species.dm:1030]: It's mostly just so it makes more sense, but I can undo it if needed.

File: fulp_modules/fulp_clothing/outfits/digi_lizard.dm (hunk 2 above)
Comment (CyoTheProtogen) [reply fulp_modules/fulp_clothing/outfits/digi_lizard.dm:99]: fixed

**Comment by CyoTheProtogen** [comment]
It actually fucking works now...

File: code/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (Helianthus00) [reply code/modules/mob/living/carbon/human/species.dm:1030]: adding a change like this will cause possible conflicts in the future and it's just for some slight flavor. Please remove it.

File: fulp_modules/fulp_clothing/digitigrade/readme.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Ports over Digigrade Shoes from the old repo!
+
+### TG Proc Changes:
+
+- None
+
+### TG files changed:
+
+- code\modules\mob\living\carbon\human\species.dm > What makes the shoes work, also some text changes.
```
Comment (JohnFulpWillard) [inline fulp_modules/fulp_clothing/digitigrade/readme.md:15]: You should remove the mention of some text change since it's been reverted
